desktop/menu.ts

```diff
@@ -0,0 +1,77 @@
+import { app, Menu, MenuItemConstructorOptions } from 'electron'
+
+const isMac = process.platform === 'darwin'
+
+const template = [
+    ...(isMac ? [{
+        label: app.name,
+        submenu: [
+            { role: 'about' },
+            { type: 'separator' },
```

Comment:
@romicolman checking the rendering in mac. I didn't add the services menu item. It should be here in the code. I am guessing there must be a way of removing it. :grimacing:

Reply:
@romicolman [Here it is mentioned in the docs](https://www.electronjs.org/docs/latest/api/menu#standard-menus) that the Services Menu is a special menu for Mac only